### PR TITLE
feature/GAL-602

### PR DESCRIPTION
**Pull request by Abdul-NHS**
https://nhsd-jira.digital.nhs.uk/browse/GAL-602

Added error banner
The button should not take the user to the summary screen if there are "0" people and the error message.

## Type of changes

- [x] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by Amar-NHS — COMMENTED** [review]
just few things to refactor. please see the comments

File: src/app/views/clinic_information/ClinicInformationPage.jsx [hunk 1]
```diff
@@ -65,16 +68,14 @@ export default function ClinicInformationPage(props) {
                 Go back
               </a>
             </div>
-          </div>
-          <div>
```
[the remark this answers is not in the document]
Comment (Abdul-NHS) [reply src/app/views/clinic_information/ClinicInformationPage.jsx:69]: Going to implement it in GAL-600 in a reusable format

File: src/app/views/clinic_information/ErrorBannerDetail.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <h2
+        class="nhsuk-error-summary__title nhsuk-u-margin-bottom-2"
+        id="error-target-title"
+      >
+        There is a problem
+      </h2>
+      <div class="nhsuk-error-summary__body">
+        <a
+          href="#error-message"
+        >
```
[the remark this answers is not in the document]
Comment (Abdul-NHS) [reply src/app/views/clinic_information/ErrorBannerDetail.jsx:26]: resolving this in GAL-600 ticket as well, where the border will be available

**Review by Amar-NHS — APPROVED** [review]
LGTM

**Review by zainm97 — APPROVED** [review]

**Review by Amar-NHS — APPROVED** [review]

**Review by Amar-NHS — APPROVED** [review]
Approve

**Comment by Abdul-NHS** [comment]
issues with sonar cloud picking up code changes that have been removed, will recreate PR

**Pull request closed without merging**